### ISSUE TITLE
NSS try to create path, filename containing invalid windows characters like colon

### ISSUE THREAD

**Issue by bourgeoa** [solid/node-solid-server#1536]
@timbl @michielbdejong @RubenVerborgh @highlevellogic

Trying NSS on a windows pod `https://bourgeoa.solid.hll.nu:7443` . Tested the `chat with me` from OS solid
The function returns a PUT error 500 
> FAILED to save new chat channel at: https://bourgeoa.solid.hll.nu:7443/IndividualChats/bourgeoa.solid.hll.nu:7443/index.ttl : Web error: 500 (Internal Server Error) on PUT of <https://bourgeoa.solid.hll.nu:7443/IndividualChats/bourgeoa.solid.hll.nu:7443/index.ttl>

This is due to the colon `:` in the folderPath

To ensure solid compatibility can we forbid the windows forbidden characters in pathName like `\/:*%"<>|` ?
This could be done in resource-mapper.js

**Comment by bourgeoa** [comment]
@jeff-zucker this may have some impact on solid-rest

**Comment by jeff-zucker** [comment]
Yes, solid-rest needs to be able to translate requests for valid URLs into valid file names on whatever the backend is.

**Comment by highlevellogic** [comment]
Is it necessary for the port to be included in the directory name?
What problem would be caused by creating chat directories like account directories, without the port?

**Comment by bourgeoa** [comment]
@highlevellogic see PR https://github.com/solid/chat-pane/pull/50
